### PR TITLE
Bump avro to 1.11.3

### PR DESCRIPTION
**Pull request by li-boxuan**
See CVE-2023-39410 for vulnerablity in avro 1.11.2 and below

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there an issue associated with this PR? Is it referenced in the commit message?
- [ ] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [ ] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -768,7 +769,7 @@
             <dependency>
                 <groupId>org.apache.avro</groupId>
                 <artifactId>avro</artifactId>
-                <version>1.11.1</version>
+                <version>${avro.version}</version>
```
Comment (FlorianHockmann) [inline pom.xml:772]: Really not important, but is there a specific reason for using a property for the version number here?

**Review by FlorianHockmann — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (li-boxuan) [reply pom.xml:772]: Ah there's no specific reason. I just thought using a property would be the standard approach?

**Comment by janusgraph-automations** [comment]
## 💔 All backports failed

| Status | Branch | Result |
|:------:|:------:|:------|
|❌|v0.6|**Backport failed because of merge conflicts**<br><br>You might need to backport the following PRs to v0.6:<br> - [Remove dangling netty dependency management](https://github.com/JanusGraph/janusgraph/pull/4023)<br> - [Bump testcontainers.version from 1.19.0 to 1.19.1](https://github.com/JanusGraph/janusgraph/pull/4019)|

### Manual backport
To create the backport manually run:
```
backport --pr 4024
```

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport) and see the [Github Action logs](https://github.com/JanusGraph/janusgraph/actions/runs/6414078456) for details

**Comment by li-boxuan** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|v0.6|[](https://github.com/JanusGraph/janusgraph/pull/4030)|

Note: Successful backport PRs will be merged automatically after passing CI.

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**